manifests/init.pp

```diff
@@ -55,6 +61,8 @@
   Stdlib::Absolutepath $webserver_static_dir = '/var/lib/pulp/docroot',
   String $postgresql_db_name = 'pulpcore',
   String $postgresql_db_password = extlib::cache_data('pulpcore_cache_data', 'db_password', extlib::random_password(32)),
+  String $django_secret_key = extlib::cache_data('pulpcore_cache_data', 'secret_key', extlib::random_password(32)),
+  Integer[0, 15] $redis_db_number = 8,
```

Comment:
I'd remove the upper limit since Redis can be configured to support more than 16 databases. The warning in the documentation is fine to keep and IMHO sufficient.

Reply:
@ekohl ok, that works for me.